.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+## Summary:
+- **What** did I do?
+- **Why** *** did I do it? (What was the motivation?)
+- **How** *** can someone test or demo my changes?
+
+  _*** **When relevant, use screenshots to document**_
+
+
+### Fixes or Addresses Issue \#: [ticket number & link]
+
+
+
+## Type of change:
+
+_Please delete options that are not relevant._
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
```

Comment:
🔧 : I dont think we need this section since we defined in the README to prefix with the type of change so this may be redundant. There is an ability in github to fail a PR right away if it doesn't have a specific pre-fix which we can look into as an option if we'd like in the future

Reply:
You're right, this is overkill now, but I do think it'd be good to add the `change/` prefix, in case we ever come into that scenario in the future. The PR rule you mentioned sounds like a good action item for the future.